### ISSUE TITLE
Delayed tagging

### ISSUE THREAD

**Issue by host6** [scm4j/scm4j-releaser#2]
- `--delayed-tag` cli option avoids root component tagging and version bumpings (will be only built)
- `delayed-tags.yml` keeps revisions and versions to be tagged. Located in current directory
- `tag` command uses `delayed-tags.yml` to tag last build

Exception is thrown if a component is building with `--delayed-tag` option and `delayed-tags.yml` already contains a dealyed tag for this component

`delayed-tags.yml` file content example
```yaml
https://com.github.myrepo/myproject
  version: 2.59.0
  revision: df7d6683c77f214a3bcde1481574362491efc28a
```

**Comment by host6** [comment]
Done.
Note: `tag` command is used.

**Comment by vitkud** [comment]
fix version bumping, make this option only for root component

**Comment by host6** [comment]
Done

**Comment by vitkud** [comment]
Do version bumping for root component only after "tag" command

**Comment by host6** [comment]
Fixed in #42
